health-check.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+check_api(){
+    DURATION=$(</dev/stdin)
+    if (($DURATION <= 5000 )); then
+        exit 60
+    else
+        curl --silent --fail "itchysats.embassy:8000/api/alive" &>/dev/null
```

Comment:
does this work? I think this script runs inside the container, so `itchysats.embassy` might not be known? Maybe it's sufficient to change to `localhost:8000/api/alive`.

Reply:
It works - the check passes and other services also define it like this :)